### DISCUSSION TITLE
Request to have @blade directives in colour

### DISCUSSION THREAD

**Discussion by eugenefvdm** [spatie/ray#154] [Ideas]
Apologies if this isn't the place to make a suggestion, and please direct me if I'm wrong.

It would be wonderful if the  @blade directive could also spit out colour.

**Comment by freekmurze** [comment]
Because not too many people are asking for this, and we don't want to maintain directives for all different colours, we're going to pass on this idea for now.

Meanwhile you could to

```
@php(ray('something')->red())
```
